css/style.css

```diff
@@ -0,0 +1,307 @@
+}
+
+h2{
+    color: var(--white);
+    font-weight:var(--fw-900);
+    font-size:var(--fs-24px);
+}
+
+h3 {
+    color: var(--white);
+    font-weight:var(--fw-400);
+    font-size: var(--fs-20px);
+}
+
+p{
+    color: var(--white);
+    font-weight:var(--fw-400);
+    font-size: var(--fs-16px);
+}
+
+button{
+    color: var(--white);
+    font-weight:var(--fw-700);
+    font-size: var(--fs-12px);
+    background-color: var(--wine);
+    border-radius: 2rem;
+    width: 140px;
+    padding:4px 0.40rem;
+}
+
+
+
+
+.home {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-auto-rows: auto;
+    grid-template-areas:
+
+    
+    "h   h   h   n"
+    "w   w   w   w"
+    "hr  hr  hr  hr"
+    "pr  pr  pr  pr"
+    "ft  ft  ft  ft";
+
+}
+
+/* HEADER */
+.home-header {
+    display: flex;
+    align-items: center;
+    grid-area: h;
+    padding:0 1.625rem;
+    background-color:var(--white);
+    height: 100px;
+}
+
+/* NAV */
+.home-nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding:0 1.625rem;
+    align-items: center;
+    grid-area: n;
+}
+.home-nav-links{
+    display: none;
+}
+
+/* WELCOME  */
+.home-welcome{
+
+    padding: 1.625rem;
+    grid-area: w;  
+    background-color:var(--pink-light)
+}
+
+.home-welcome img{
+    height: 47px;   
+}
+
+.home-welcome-section1{
+   padding-bottom: 16px; 
+}
+
+.home-welcome-section2{
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 10px;
+}
+
+.home-welcome-section2--img-tablet{
+    display: none;
+}
+
+.delivery{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+}
+.delivery > p{
+    padding-bottom: 7px;
+    letter-spacing: 0.03rem;
+}
+
+/* HERO */
+.home-hero{
+    grid-area: hr;
+}
+.home-hero img{
+    width: 100%;
+}
+/* PRODUCTS */
+.products {
+    display: flex;
+    grid-area: pr;
+    justify-content: center; 
+}
+.products-cards h2{
+    text-align: center;
+    padding:1.625rem; 
+    color: var(--wine);
+}
+
+.products img {
+    width: 158px;
+    margin: 8px;
+}
+.products-cards-items{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+/* FOOTER */
+.home-footer {
+    margin-top: 30px;
+    padding-top: 30px;
+    padding-bottom: 100px;
+    text-align: center;
+    grid-area: ft;
+    background-color:var(--cream-light);
+
+}
+
+/* MEDIA QUERIES */
+
+/* Tablet Version */
+@media only screen and (min-width:540px)  {
+    .home {
+    justify-content: center;
+    grid-template-columns:repeat(6, 1fr);
+    grid-template-areas:
+
+    "h    h    h    h    h    h    h   h"
+    "n    n    n    n    n    n    n   n"
+    "hr   hr   hr   w    w    w    w   w"
+    "pr   pr   pr   pr   pr   pr   pr  pr"
+    "ft   ft   ft   ft   ft   ft   ft  ft"; 
+    }
+/* NAV */
+    .home-nav {
+        display: flex;
+        justify-content: center;
+        height: 50px;
+        grid-area: n;
```

Comment:
do you need to repeat all of these properties? Remember if the values don't change, the base styles (the ones not in a media query) will be applied, so there is no need to redefine them in the media query unless they change